Add missing introductory exportPathMap purpose description (#17444)
vercel/next.js: docs/api-reference/next.config.js/exportPathMap.md

# exportPathMap

<details>
  <summary><b>Examples</b></summary>
  <ul>
    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/with-static-export">Static Export</a></li>
  </ul>
</details>

> > This feature is exclusive of `next export`. Please refer to [Static HTML export](/docs/advanced-features/static-html-export.md) if you want to learn more about it.

Reviewer: This note should stay on the top of the page.

# exportPathMap

> `exportPathMap` allows you to specify a mapping of request paths to page destinations, to be used during export.

Reviewer: ```suggestion
`exportPathMap` allows you to specify a mapping of request paths to page destinations, only used by `next export`.
```

Reviewer: Also can you move this to be under the example, not on top of the page 🙏 

All of our pages show the examples first. That may change in the future but for the moment it should keep being consistent.
